### PR TITLE
full audit and rename

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2d9550a055/how-to/)

File: content/general/when-is-managed-colocation-a-good-fit.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-As a business leader, it's important for you to determine which solution works best for your business's strategy and needs. [Managed Colocation](https://www.rackspace.com/dedicated-servers/colocation), one such infrastructure option, is the hosting choice of many businesses because it provides the desired level of control while offloading some of the inconvenience, financial risk, and upfront capital.
+As a business leader, it's important for you to determine which solution works best for your business's strategy and needs. [Managed Colocation](https://www.rackspace.com/dedicated-servers/colocation), one such infrastructure option, is the hosting choice of many businesses because it provides the desired level of control while offloading some of the inconvenience, financial risk, and upfront capital. Managed Colocation means that businesses don't need to fund, build, or maintain their own data centers, and the business is not responsible for creating their own high performance, robust network. Instead business who chose Managed Colocation take advantage of Rackpsace's professional facilities, and networks and have access to other services and managed solutions.
```
Comment (stephamon) [inline content/general/when-is-managed-colocation-a-good-fit.md]: Last sentence: "Instead, businesses who choose Managed Colocation take advantage..."

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-61036925d8/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**